tuneme/static/new/js/main.js

```diff
@@ -3,9 +3,17 @@
   var domReady = function(callback) {
       document.readyState === "interactive" || document.readyState === "complete" ? callback() : document.addEventListener("DOMContentLoaded", callback);
   };
+
   domReady(function() {
     var selectRadius = document.getElementById('select_radius');
-    console.log(selectRadius[selectRadius.selectedIndex].value);
+    if(selectRadius){
+        console.log(selectRadius[selectRadius.selectedIndex].value);
+        // get selected option in sel (reference obtained above)
+        var optFunc = getSelectedOption(selectRadius);
+        selectRadius.addEventListener('click', function() {
+          console.log( optFunc.value," - ", optFunc.text);
+        });
```

Comment:
@Mitso is this piece of JS still relevant?

Reply:
@moh-moola Please remove 👍